packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -565,7 +565,7 @@ const AccountSwitcher = props => {
                                     <div key={account.title} className='acc-switcher__new-account'>
                                         <Icon icon={`IcMt5-${account.icon}`} size={24} />
                                         <Text size='xs' color='general' className='acc-switcher__new-account-text'>
-                                            {account.title}
+                                            {getFormattedAccountTitle(account.title)}
```

Comment:
Is the title coming from BE? If so, won't BE change the title to `Derived`?

Reply:
We have places where it manually changes the title on FE side, but I realized this function is not needed anymore, since I changed the title in those places already, so I remove it.